archives/2020/student-reports/port-Sugar-and-core-activities-to-Python-3-Saumya-Mishra.md

```diff
@@ -13,10 +13,10 @@ One of the main aims of the project was to review the Sugar source code changes
 
 ### Experience
 
-It has been a stunning experience until now. It helped me learning techniques of code debugging, optimization, maintenance, and user experience. It taught me the value of effective communication. I would especially like to mention James for helping throughout the period with his constructive feedback and honest reviews of my work which have proved to be a bar raiser for me. Those feedbacks encouraged me to be more focused while working and write quality code. Also, Ibiam helped throughout the journey from 1st day till the end by his great reviews. Rahul, he helped me to remain on track every time and to make sure things are going well.
+It has been a stunning experience overall. It has helped me in learning techniques of code-debugging, optimization, maintenance, and user experience. It taught me the value of effective communication. I would especially like to mention James for helping throughout the period with his constructive feedback and honest reviews of my work which have proved to be a bar-raiser for me. Those feedbacks encouraged me to be more focused while working and write quality code. Also, Ibiam helped throughout the journey from 1st day till the end by his great reviews. Rahul, he helped me to remain on track every time and to make sure things are going well.
```

Comment:
`It has taught me` you can make this change also

Reply:
ohkk